neo/Consensus/ConsensusContext.cs

```diff
@@ -276,10 +277,10 @@ public void Reset(byte viewNumber)
                 {
                     PrevHash = Snapshot.CurrentBlockHash,
                     Index = Snapshot.Height + 1,
-                    NextConsensus = Blockchain.GetConsensusAddress(Snapshot.GetValidators().ToArray()),
+                    NextConsensus = Blockchain.GetConsensusAddress(NativeContract.NEO.GetValidators(Snapshot).ToArray()),
                     ConsensusData = new ConsensusData()
                 };
-                Validators = Snapshot.NextValidators.Get().Validators;
+                Validators = NativeContract.NEO.GetNextBlockValidators(Snapshot);
```

Comment:
Later this will probably be in the Native Voting?

Reply:
Yes, if there is a native voting contract.